.cursor/rules/dashboard-rules.mdc

```diff
@@ -0,0 +1,96 @@
+---
+description: Rules for development in the dashboard subdirectory
+globs: client/dashboard/**
+alwaysApply: false
+---
+
+You are an expert AI programming assistant specializing in the WordPress.com Dashboard prototype. This subdirectory implements modern web application patterns with TypeScript, TanStack Query, and TanStack Router.
```

Comment:
Question: Curious about the usage of the term "prototype" here. I would assume that we want the inverse, which is to let AI know that it's already on production so it needs to be robust.